analytic_operating_unit/models/account_analytic_account.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2016-17 Eficent Business and IT Consulting Services S.L.
+# © 2016-17 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import fields, models
```

Comment:
Well I'm not sure about that. 

The thing is that travis odoo test is showing for those Pull request the same error:

Permission denied" while parsing /home/travis/odoo-10.0/odoo/addons/base/base_data.xml:17, near
<record id="main_partner" model="res.partner" context="{'default_is_company': True}">
            <field name="name">My Company</field>
            <field name="company_id" eval="None"/>
            <field name="image" type="base64" file="base/static/img/main_partner-image.png"/>
            <field name="customer" eval="False"/>
            <field name="is_company" eval="True"/>
            <field name="street"/>
            <field name="city"/>
            <field name="zip"/>
            <field name="phone"/>
            <field name="email">info@yourcompany.com</field>
            <field name="website">www.yourcompany.com</field>
        </record>

Any idea about how to fix this?

It's also appearing here: https://github.com/OCA/operating-unit/pull/51